striprtf/striprtf.py

```diff
@@ -192,11 +191,11 @@ def rtf_to_text(text, encoding="utf-8", errors="strict"):
                     if c < 0:
                         c += 0x10000
                     try:
-                        chr1 = chr(c).encode(encoding, errors).decode(encoding).encode('utf-8', errors)
+                        chr1 = chr(c).encode(encoding, errors).decode(encoding).encode('utf-8', errors).decode('utf-8')
                     # because some flavors of rtf seem to allow utf8 inside of cp1252
                     except UnicodeEncodeError as e:
-                        chr1 = chr(c).encode('utf-8', errors)
-                    out += chr1.decode('utf-8')
+                        chr1 = chr(c)
+                    out += chr1
```

Comment:
Why can't you delete the `try` block and the `chr1` variables completely?  I think you only need to do:
```
out += chr(c)
```

Reply:
(The point is that the `encoding` "code page" is *only* used for hex directives `\'xx`.  `\u` directives specify Unicode codepoints explicitly, which mostly don't exist in the old "code page" encodings, and you don't want to go through `encode` at all.  Code pages are pre-Unicode baggage that should be avoided as much as possible.)

Reply:
see new pr https://github.com/joshy/striprtf/pull/43
trying to do this while doing my "real" job is slowing me down, thank you very much for the help and support, I am learning a lot more about RTF/UTF than I ever wanted....